exonum-java-binding/tests_profile

```diff
@@ -49,17 +49,14 @@ export JAVA_LIB_DIR="$(find ${JAVA_HOME} -type f -name libjvm.\* | xargs -n1 dir
 echo "JAVA_LIB_DIR=${JAVA_LIB_DIR}"
 
 # Version of Rust used to build tests.
-## Stable works well unless you want benchmarks.
-# TODO: stable does not work well until ECR-1839 is resolved
 export RUST_COMPILER_VERSION="${RUST_COMPILER_VERSION:-1.32.0}"
 echo "RUST_COMPILER_VERSION: ${RUST_COMPILER_VERSION}"
 
 # Find the directory containing Rust libstd.
 export RUST_LIB_DIR=$(rustup run ${RUST_COMPILER_VERSION} rustc --print sysroot)/lib
 echo "RUST_LIB_DIR: ${RUST_LIB_DIR}"
 
-# Linking options with additional paths for the application and tests.
-# Note: `-C link-args` can be used only once, therefore, `-C link-arg` is used.
+# Linking options with additional paths for the application.
```

Comment:
For the application or for the tests ? We seem to set for the app in the packaging script?